functions/operatingHours.ts

```diff
@@ -128,7 +166,20 @@ export const handler = async (
 
     const status = getOperatingStatus(operatingInfo, channel, office);
 
-    resolve(success(status));
+    // Support legacy function to avoid braking changes
+    // TODO: remove once every account has been migrated
+    if (!useV2) {
```

Comment:
I know it's for compatibility and we never expect a caller to explicitly NOT use V2, but it's still a bit weird that passing `'false'` into `useV2` would activate v2... 

Also, sorry to be a pain but can we maybe use a more descriptive name for this flag? Like `includeMessageTextInResponse` or something? 'V2' doesn't really help people understand what the flag is doing.